homie5/formats/boolean.yml

```
@@ -2,51 +2,74 @@ description: validating boolean formats
 
 tests:
   - description: proper format is accepted
-    type: boolean
-    format: 'off,on'
+    type: propertydescription
+    definition:
+      datatype: boolean
+      format: "off,on"
```

```suggestion
    testtype: propertydescription
    definition:
      type: boolean
      format: "off,on"
```
Can we use `type` in the definition (and then use `testtype` for the other one)? To stay closer to the homie objects.

---

I am fine with testtype but in the definition I need to use `datatype` as it is a property description.  Or did we change the spec for this?

---

sorry, I'm an idiot